### ISSUE TITLE
Pdal role lost connection to remote host during pdal build

### ISSUE THREAD

**Issue by meiertgrootes** [nlesc-sherlock/emma#139]
When building pdal from source within the pdal role, having specified the python3 executable as in #138 Ansible reports failure after 5 minutes, saying that the remote host has closed ssh connection. Logging into remote host, `which pdal` does find installed pdal.

**Comment by meiertgrootes** [comment]
just to draw your attention @fdiblen @sverhoeven

**Comment by meiertgrootes** [comment]
**Update:** @fdiblen @sverhoeven 
I ran the pdal role again, but stopping after running intial CMake. Logged into node nd checked make version (which make -> 4.1). Ran remaining steps manually without issues, resulting in pdal installation.
then retried with new cluster but including the `sudo make -j2` command. Again lost connection, with message that connection closed by host. logged in again and ran `sudo make install` , which executed without issues.
Can this be terraform/ansible timing out?
